src/Core/CoreKernel.php

```diff
@@ -498,10 +506,15 @@ protected function bootManifests($flush)
                 $config->setFlush(true);
             }
         }
+        // tell modules to sort, now that config is available
+        $this->getModuleLoader()->getManifest()->sort();
```

Comment:
Looks like this will impact the order in which `_config.php` files are loaded when `bootConfigs()` is called. This will mean that `$project/_config.php` will be loaded earlier if it’s a higher priority (which it usually will be). Should this order be reversed for `_config.php` files? e.g. if I set my module to have a higher priority than `framework`, I can override templates but I can’t override the `embed` shortcode parser